.github/workflows/build-rust-cross-platform.yml

```diff
@@ -39,13 +45,29 @@ jobs:
       - name: Cache cargo registry
         uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84 # v2.7.3
 
+      - uses: goto-bus-stop/setup-zig@7ab2955eb728f5440978d5824358023be3a2802d # v2.2.0
+        if: ${{ contains(matrix.settings.target, 'musl') }}
+        with:
+          version: 0.12.0
+
+      - name: Install Zigbuild
+        if: ${{ contains(matrix.settings.target, 'musl') }}
+        run: cargo install cargo-zigbuild --locked --git https://github.com/rust-cross/cargo-zigbuild --rev 6f7e1336c9cd13cf1b3704f93c40fcf84caaed6b # 0.18.4
+
       - name: Add build architecture
         run: rustup target add ${{ matrix.settings.target }}
 
-      - name: Build Rust
+      - name: Build Rust for - ${{ matrix.settings.target }}
+        if: ${{ contains(matrix.settings.target, 'musl') }}
+        env:
+          RUSTFLAGS: "-D warnings"
+        run: cargo zigbuild -p bitwarden-c --target ${{ matrix.settings.target }} --release
+
+      - name: Build Rust for - ${{ matrix.settings.target }}
+        if: ${{ !contains(matrix.settings.target, 'musl') }}
         env:
           RUSTFLAGS: "-D warnings"
-        run: cargo build --target ${{ matrix.settings.target }} --release
+        run: cargo build -p bitwarden-c --target ${{ matrix.settings.target }} --release
```

Comment:
I adjusted this workflow so that it _only_ builds the artifacts for the `bitwarden-c` crate. Let me know if the other builds are actually necessary in this workflow. As far as I can tell, they aren't getting used anywhere else, but I could have missed something

Reply:
Yeah, we are only uploading artifacts for files matching `*bitwarden_c*`, so this seems fine to me.